src/test/resources/generic/payers/test_attributes_eligibility.csv

```diff
@@ -0,0 +1,8 @@
+attributes
+test1 = true
```

Comment:
Is `test1 = true` assignment or an equality check?

Reply:
it's an equality check
It's set so a user could use either == or = to try and keep it user friendly.